### PR TITLE
Always print manifest versions in workload list command

### PR DESCRIPTION
**Pull request by Forgind**

### CODE REVIEW

File: src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs [hunk 1]
```diff
@@ -105,15 +105,12 @@ public override int Execute()
...
+                    table.AddColumn(InformationStrings.WorkloadManifestVersionColumn, workload =>
                     {
-                        table.AddColumn(InformationStrings.WorkloadManifestVersionColumn, workload =>
-                        {
-                            var m = _workloadListHelper.WorkloadResolver.GetManifestFromWorkload(new WorkloadId(workload.Key));
-                            var manifestInfo = manifestInfoDict[m.Id];
-                            return m.Version + "/" + manifestInfo.ManifestFeatureBand;
-                        });
-                    }
+                        var m = _workloadListHelper.WorkloadResolver.GetManifestFromWorkload(new WorkloadId(workload.Key));
```
Comment (joeloff) [inline src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs:110]: Any chance this can through key not found exceptions or return null?

Comment (Forgind) [reply src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs:110]: That would mean we'd installed a workload but didn't install its associated manifests, which would mean we have a bug in the WorkloadInstallCommand.

Also, it's moved around a little, but this call has existed for years:
https://github.com/dotnet/sdk/blame/4acb5698e77cd7d6dfee3e4059faae94f6bbe7e8/src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs#L90

**Comment by marcpopMSFT** [comment]
Can you add a screenshot of the after behavior and what does it do if there's not enough horizontal space?

**Review by marcpopMSFT — APPROVED** [review]

**Pull request merged**